packages/fast-usdc/src/utils/make-helpers.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Takes an object of make* functions and returns an object with a single 'make' property
+ * containing those functions renamed without the 'make' prefix.
+ *
+ * @template {Record<`make${string}`, Function>} T
+ * @param {T} makers - Object containing make* functions
+ * @returns {{ make: { [K in keyof T as K extends `make${infer R}` ? R : never]: T[K] } }} Transformed object
+ */
+export const organizeMakers = makers => {
+  const make = {};
+  for (const [key, fn] of Object.entries(makers)) {
+    if (!key.startsWith('make')) {
+      throw new Error(`Unexpected key ${key}`);
+    }
+    const newKey = key.slice(4); // Remove 'make'
+    make[newKey] = fn;
+  }
+  // @ts-expect-error cast
+  return harden({ make });
```

Comment:
why a record? why not `return make;`? in case there are other facets later or something? that doesn't seem like your style.

Reply:
To enforce that the name used is `make`. Though that could be done by a lint rule, like React's `useState` does to enforce that the tuple it returns of [state object, state setter] has the the consistent naming `[myState, setMyState]`.